index.html

```diff
@@ -0,0 +1,461 @@
+                            </svg>                            
+                    </div>
+                    <div class="card__name">
+                        Digital Marketing
+                    </div>
+                </div>
+                <p class="card__desc">
+                    the quick fox jumps over the lazy<br />
+                    dog the quick fox jumps over the <br />
+                    lazy dog
+                </p>
+            </div>
+            <div class="card__item">
+                <div class="card__header">
+                    <div class="card__img">
+                        <svg width="48" height="41" viewBox="0 0 48 41" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path d="M42.5886 8.52554H37.519V3.45592C37.519 2.55955 37.1629 1.6999 36.5291 1.06608C35.8952 0.432251 35.0356 0.0761719 34.1392 0.0761719H13.8608C12.9644 0.0761719 12.1047 0.432251 11.4709 1.06608C10.8371 1.6999 10.481 2.55955 10.481 3.45592V8.52554H5.41139C4.06684 8.52554 2.77736 9.05966 1.82662 10.0104C0.875886 10.9611 0.341766 12.2506 0.341766 13.5952V37.2534C0.341766 38.1497 0.697846 39.0094 1.33167 39.6432C1.9655 40.2771 2.82515 40.6331 3.72151 40.6331H44.2785C45.1748 40.6331 46.0345 40.2771 46.6683 39.6432C47.3021 39.0094 47.6582 38.1497 47.6582 37.2534V13.5952C47.6582 12.2506 47.1241 10.9611 46.1734 10.0104C45.2226 9.05966 43.9331 8.52554 42.5886 8.52554ZM13.8608 3.45592H34.1392V8.52554H13.8608V3.45592ZM44.2785 37.2534H3.72151V22.0445H17.2405V30.4939H30.7595V22.0445H44.2785V37.2534ZM20.6202 22.0445H27.3797V27.1141H20.6202V22.0445ZM3.72151 18.6648V13.5952C3.72151 13.147 3.89955 12.7172 4.21647 12.4002C4.53338 12.0833 4.9632 11.9053 5.41139 11.9053H42.5886C43.0368 11.9053 43.4666 12.0833 43.7835 12.4002C44.1004 12.7172 44.2785 13.147 44.2785 13.5952V18.6648H3.72151Z" fill="#00A0C1"/>
+                            </svg>                            
+                    </div>
+                    <div class="card__name">
+                        Digital Marketing
+                    </div>
+                </div>
+                <p class="card__desc">
+                    the quick fox jumps over the lazy<br />
+                    dog the quick fox jumps over the <br />
+                    lazy dog
+                </p>
+            </div>
+        </div>
+    </section>
+    <section class="service">
+       <img class="service__img" src="./img/Features-11.png" />
+    </section>
+    <section class="fqs">
```

Comment:
I don't think the name "fqs" is clear enough.... I have no idea what it's referring to.